src/stores/Tables/Billing/Hydrator.tsx

```diff
@@ -32,9 +35,11 @@ export const BillingHistoryTableHydrator = ({
 
     const hydrateState = useBillingTable_hydrateContinuously();
 
-    const { billingHistory, error, isValidating } = useBillingHistory({
-        query,
-    });
+    const {
+        billingRecord: billingHistory,
+        error,
+        isValidating,
+    } = useBillingRecord('');
```

Comment:
Is this call why we handle when a `date` or a `string` is passed to this hook?

Reply:
No, it is because time can be represented as a `string` or a `Date` and I wanted the query builder to handle both cases.